app/services/provider_interface/application_choice_presenter.rb

```diff
@@ -24,8 +25,12 @@ def to_param
       application_choice.id
     end
 
+    def date_of_birth
```

Comment:
For future consideration: the duplication with `ApplicationFormPresenter` bugs me, we perhaps need to come up with a better way to share code between support & provider.

Reply:
The duplication doesn't seem to be getting out of hand just yet, seems better to me to stick with some duplication rather than try to create some generic mixin/base class.

Using `delegate` would I guess reduce the number of lines of code duplicated a little but doesn't really address the issue and you might prefer explicit methods for `date_of_birth` etc.